tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+      summary: Delete multiple subscriptions
+      description: Deletes one or more subscriptions you specify. This route is deprecated in favor of /v2/delete-subscriptions.
+      operationId: RpcDeleteSubscriptions
+      deprecated: true
+      parameters:
+        - in: body
+          name: subscriptionIds
+          description: IDs of the subscriptions whose heartbeat timers should be updated
+          required: false
+          schema:
+            type: object
+            properties:
+              subscriptionIds:
+                type: array
+                items:
+                  type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Response for deleting subscriptions
+            type: object
+            title: Delete Subscriptions Response
+            properties:
+              invalidSubscriptionIds:
+                description: List of invalid subscription IDs
+                type: array
+                items:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/delete-subscriptions:
+    post:
+      tags:
+        - subscriptions
+      summary: Delete multiple subscriptions (SystemLink Server only)
+      description: Delete one or more subscriptions based on the request body.
+      operationId: DeleteSubscriptions
+      parameters:
+        - in: body
+          name: subscriptionIds
+          required: false
+          schema:
+            type: object
+            properties:
+              subscriptionIds:
+                type: array
+                items:
+                  type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Response for deleting subscriptions
+            type: object
+            title: Delete Subscriptions Response
+            properties:
+              invalidSubscriptionIds:
+                description: List of invalid subscription IDs
+                type: array
+                items:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections:
+    post:
+      tags:
+        - selections
+      summary: Create a selection
+      description: Selects a collection of tags for subsequent operations. Selections expire after up to fifteen minutes of inactivity.
+      operationId: CreateSelection
+      parameters:
+        - in: body
+          name: selection
+          description: Selection to create
+          schema:
+            $ref: '#/definitions/Selection'
+          required: false
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/Selection'
+        400:
+          description: Bad Request
+        401:
+          $ref: '#/responses/Unauthorized'
+        409:
```

Comment:
Kind of like 404, SL Server returns an error response in this case.

Reply:
Removed the 409s